Formula/podman.rb

```diff
@@ -28,6 +28,6 @@ def install
 
   test do
     assert_match "podman version #{version}", shell_output("#{bin}/podman -v")
-    assert_match "Error: Get \"http://d/v1.0.0/libpod../../../_ping\"", shell_output("#{bin}/podman 2>&1", 125)
+    assert_match "Error: Get", shell_output("#{bin}/podman 2>&1", 125)
```

Comment:
```suggestion
    assert_match "Error: Get", shell_output("#{bin}/podman info 2>&1", 125)
```

Reply:
Thank you for your suggestions @ashley-cui. I applied them without looking in to it because I did not have much time.

Reply:
No problem! Glad to help